clients/auth/mon-compte-pro/strategy.ts

```diff
@@ -40,11 +41,26 @@ export const getClient = async () => {
 export const monCompteProAuthorizeUrl = async () => {
   const client = await getClient();
   return client.authorizationUrl({
-    scope: 'openid email',
+    scope: 'openid email organizations profile',
   });
 };
 
-export const monCompteProGetToken = async (req: any) => {
+export type IMCPUserInfo = {
+  email: string;
+  email_verified: boolean;
+  family_name: string;
+  given_name: string;
+  organizations: {
+    id: number;
+    siret: string;
+    is_external: boolean;
+    label: string;
+    is_collectivite_territoriale: boolean;
+    is_service_public: boolean;
+  }[];
+};
```

Comment:
Voici la payload complète :

```suggestion
export type IMCPUserInfo = {
  sub: string;
  email: string;
  email_verified: boolean;
  family_name: string | null;
  given_name: string | null;
  phone_number: string | null;
  job: string | null;
  organizations: {
    id: number;
    siret: string;
    is_external: boolean;
    label: string | null;
    is_collectivite_territoriale: boolean;
    is_service_public: boolean;
  }[];
};
```

Sachant que phone_number va peut être disparaitre prochainement.

À savoir aussi que given_name, family_name, phone_number et job sont des données obligatoires pour l'inscription. Donc si une personne arrive avec un de ces champs à null, c'est qu'il y a un bug dans MonComptePro.